tasks/save-collectors-results/tests/test-save-collectors-results-not-valid-name.yaml

```diff
@@ -0,0 +1,90 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-save-collectors-results
+spec:
+  description: |
+    Run the save-collectors-results task with rbac present and one proper results
+    file in the results directory. The pipeline should succeed.
```

Comment:
So there is one proper file, but the other has invalid name, right? And that one will be ignored. Can you add that to the description please?

Reply:
Done.